.eslintrc.js

```diff
@@ -5,6 +5,9 @@ module.exports = {
     parserOptions: {
         allowImportExportEverywhere: true,
     },
+    rules: {
+        'import/extensions': 'off',
+    },
```

Comment:
Was wondering why we need that. Typescript?

Reply:
https://github.com/dhis2/app-platform/pull/721#issuecomment-1160497608

I can take care of it after we've dealt with the functional part of these changes. Having it not block these changes makes it a little easier for me to integrate it with other tasks.